controls/pcidss_4_ocp4.yml

```diff
@@ -2120,21 +2154,42 @@ controls:
         from being performed while the console/PC is unattended.
       levels:
         - base
-      status: pending
-      rules: []
+      status: automated
+      notes: |-
+        Session timeouts can be enabled with OpenShift to limit the amount of
+        time that a session can be active. However, the payment entity also needs to control the
+        user's and administrator's idle session timeouts on their payment applications as well.
+      rules:
+        - oauth_or_oauthclient_inactivity_timeout
+        - var_oauth_inactivity_timeout=15m0s
 
   - id: '8.3'
     title: Strong authentication for users and administrators is established and managed.
     levels:
       - base
-    status: pending
+    status: not applicable
+    notes: |-
+      For this control to be satisfiable an identity provider must be used and the kubeadmin user
+      needs to be removed.
+    rules:
+      - idp_is_configured
+      - kubeadmin_removed
```

Comment:
Does rule `ocp4-ocp-idp-no-htpasswd` need to be added here as the tile said "Strong authentication for users and administrators is established and managed"

Reply:
What about the rule "ocp_no_ldap_insecure"?

Reply:
+1
I think those two rules should be added here:
  - ocp_idp_no_htpasswd
  - ocp_no_ldap_insecure

Reply:
Thanks for the reviews.

I have added `ocp_no_ldap_insecure` under `8.3.2`. the rule is about making sure LDAP doesn't transmit clear passwords.
And  have added `ocp_idp_no_htpasswd` under `8.3.4`, PCI-DSS is okay with using passwords as an authentication mechanism, and requirement `8.3.6` establishes strong parameters for this password. However, the `htpaswd` identity provider doesn't provide user lockout, which is required by `8.3.4`.